fastchat/model/model_adapter.py

```diff
@@ -626,6 +626,24 @@ def load_model(self, model_path: str, from_pretrained_kwargs: dict):
     def get_default_conv_template(self, model_path: str) -> Conversation:
         return get_conv_template("zero_shot")
 
+class ChangGPTAdapter(BaseAdapter):
+    """The model adapter for lcw99/polyglot-ko-12.8b-chang-instruct-chat"""
+
+    def match(self, model_path: str):
+        print(model_path)
+        return "polyglot" in model_path and "chang" in model_path
+
+    def load_model(self, model_path: str, from_pretrained_kwargs: dict):
```

Comment:
this function can be removed.
1. Set `use_fast_tokenizer = True` in your class, see.
2. Then this function can be removed because it is the same as the function in the base class